### ISSUE TITLE
can't make div transperant when mouse over

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#49]
```

What is the expected output? What do you see instead?

i want the div background be half transperant when using :hover (mouse over)
its hilights just the links in the div

What version of the product are you using? On what operating system?
ie8

Please provide any additional information below.

 the code i used is:


.HomeLeftCenterPatInner     { width:228px; padding:0px 4px 0px 5px;
height:96px; margin:0 auto; overflow:hidden; background-color:#000000;}

.HomeLeftCenterPatInner :hover  { width:228px; padding:0px 4px 0px 5px;
height:96px; margin:0 auto; overflow:hidden; background-color:#000000;
filter:alpha(opacity=75); -moz-opacity:0.75; opacity: 0.75;}
```

Original issue reported on code.google.com by `harell...@gmail.com` on 5 Mar 2008 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
I did managed it by changing the HTML with Onmouseover function for class change

```

Original comment by `harell...@gmail.com` on 5 Mar 2008 at 12:56

**Comment by GoogleCodeExporter** [comment]
```
This seems to be fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 2:24
- Changed state: **Fixed**
